src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,25 @@
+$color-main: #060b35;
+$color-secondary: #616070;
+$card-width: 200px;
+$card-margin: 48px;
+
+%font-400 {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: $color-secondary;
+}
+
+%font-500 {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: $color-main;
+}
+
+%font-700 {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: $color-main;
+}
```

Comment:
Move this code to a new file `_extends.scss`

More details in [this video, timing 1:40](https://mate.academy/learn/css-advanced/sass#/video/931)